### ISSUE TITLE
Need help in CustomBoxy with CustomScrollView

### ISSUE THREAD

**Issue by RaviKavaiya** [PixelToast/flutter-boxy#1]
Hi @PixelToast :wave:

Thanks for this great package. 
I am stuck at a place and need some help. Actually, I was experimenting with CustomBoxy.

The thing is, the child of my CustomBoxy is a `CustomScrollView` itself (To be clear, I am trying to show this: https://pub.dev/packages/implicitly_animated_reorderable_list).

First, let me show code:

```dart
class MyDelegate extends BoxyDelegate {
  @override
  layout() {
    final list = getChild(#list);

    final listSize = list.layout(constraints.copyWith(
      maxHeight: 250,
    ));

    list.position(Offset.zero);

    return Size(listSize.width, listSize.height);
  }
}

///
/// Somewhere...
///
return CustomBoxy(
  delegate: MyDelegate(),
  children: [
    LayoutId(
      id: #list,
      child: ImplicitlyAnimatedReorderableList<String>(
        // ... ... ...
      ),
    ),
  ],
);
```
The error I get:
```
══╡ EXCEPTION CAUGHT BY RENDERING LIBRARY ╞═════════════════════════════════════════════════════════
The following assertion was thrown during performLayout():
'package:flutter/src/widgets/framework.dart': Failed assertion: line 2598 pos 12: '!_debugBuilding':
is not true.
Either the assertion indicates an error in the framework itself, or we should provide substantially
more information in this error message to help you determine and fix the underlying cause.
In either case, please report this assertion by filing a bug on GitHub:
  https://github.com/flutter/flutter/issues/new?template=BUG.md
The relevant error-causing widget was:
  ImplicitlyAnimatedReorderableList<String>
```

#### What I wanted to do?
I wanted to achieve a similar effect of `ProductTile` example that you have provided where this list would be shown at top with a FAB at bottom.

I am pretty new to this. If this can't be done using CustomBoxy, what I should I try to do??

Thanks...

**Comment by PixelToast** [comment]
This is indeed a bug in Boxy, thank you for reporting it!

**Comment by PixelToast** [comment]
Fixed in v1.2.0, again thank you @RaviKavaiya

**Comment by RaviKavaiya** [comment]
@PixelToast That was a REALLY REALLY quick fix. It worked!! Thank you so much...:thumbsup:
